packages/notifier/test/test-publish-kit.js

```diff
@@ -670,17 +678,17 @@ const verifySubscribeAfterSequencing = test.macro(async (t, makePublishKit) => {
 
   const sub1FirstLateAll = [];
   const sub1SecondAll = [];
-  E.when(
+  void E.when(
     sub1.subscribeAfter(),
     cell => void sub1FirstLateAll.push(cell),
     t.fail,
   );
```

Comment:
Please apply the same pattern here and to the subsequent similar clauses in this file:

```suggestion
  E.when(
    sub1.subscribeAfter(),
    cell => void sub1FirstLateAll.push(cell),
  ).catch(t.fail)
```